.editorconfig

```diff
@@ -0,0 +1,23 @@
+root = true
+
+[*.{py,toml,yml,md}]
+indent_style = space
+end_of_line = lf
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.py]
+max_line_length = 88
+indent_size = 4
+
+[*.toml]
+max_line_length = 150
+indent_size = 4
+
+[*.yml]
+max_line_length = 120
+indent_size = 2
```

Comment:
I'd cut these two. Right now we rarely use either kind of file and both are unformatted.

Reply:
This is for editors, not formatters.  The indent size tells the editor how many spaces to add when <tab> is pressed.    I got rid of the line length then.